docs/getting-help.md

```diff
@@ -0,0 +1,43 @@
+---
+tags:
+ - Getting Help
+---
+
+# Getting Help
+
+## Additional Documentation
+
+If you haven't found your answer elsewhere in these pages, your answer may be in the documentation for the system you are using:
+
+- [Engaging Documentation](https://engaging-web.mit.edu/eofe-wiki/)
+- [Satori Documentation](https://mit-satori.github.io/)
+- [SuperCloud Documentation](https://supercloud.mit.edu/)
+- [OpenMind Documentation](https://github.mit.edu/MGHPCC/OpenMind/wiki)
+
+## Email
+
+If you can't find your answer in the documentation, please use one of the email lists below to contact us. With the exception of SuperCloud, these lists will create a ticket that our team can assign and track. In all cases these mailing lists includes the entire team, so the best available person to answer your question will respond. Sending email to the entire team will also likely get you the fastest response. Please do not send email directly to individual team members.
+
+- General ORCD Questions: [orcd-help at mit.edu](mailto:orcd-help@mit.edu)
+- Engaging: [orcd-help-engaging at mit.edu](mailto:orcd-help-engaging@mit.edu)
+- Satori: [orcd-help-satori at mit.edu](mailto:orcd-help-satori@mit.edu)
+- OpenMind: [orcd-help-openmind at mit.edu](mailto:orcd-help-openmind@mit.edu)
+- SuperCloud: [supercloud at mit.edu](mailto:supercloud@mit.edu)
+
+In this email, please provide, where applicable:
+
+- Description of your issue or request
+- The command that you used to launch your job
+- Job ID(s)
+- What you tried
+- The full error message you are receiving
+- Any supporting files (code, submission scripts, screenshots, etc)
+
+## Office Hours
+
+We host weekly office hours. Office Hours are a time when you can drop in and ask us questions. It is a great time to discuss or troubleshoot something that is difficult over email. See the table below for the available Office Hours sessions.
+
+| Sesison | Time | Location |
+| ----------- | ----------- |----------- |
+| In Person Office Hours  | Thursdays 2-3 pm | Rotch Library GIS and Data Lab |
```

Comment:
Maybe add links 
[Rotch Library ](https://maps.google.com/maps?hl=en&q=Rotch%20Library%2C%2077%20Massachusetts%20Ave%2C%20Cambridge%2C%20MA%2002139%2C%20USA)

and

[GIS and Data Lab](https://libraries.mit.edu/data-services/gis-data-lab/)